src/Pact/Eval.hs

```diff
@@ -910,7 +1008,7 @@ resumePactExec i req ctx = do
 appError :: Info -> Doc -> Term n
 appError i errDoc = TApp (App (msg errDoc) [] i) i
 
-resolveFreeVars ::  Info -> Scope d Term Name ->  Eval e (Scope d Term Ref)
+resolveFreeVars ::  Info -> Scope Int Term Name -> Eval e (Scope Int Term Ref)
```

Comment:
Huh .. did I make this change? I'm thinking leaving it unspecialized is good from a legibility point of view ("this function doesn't care about the bound param")